### DISCUSSION TITLE
Obsidian seems to be stuck at v1.3.5.

### DISCUSSION THREAD

**Discussion by malikalimoekhamedov** [Homebrew/discussions#4644] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.28
ORIGIN: https://github.com/Homebrew/brew
HEAD: 2b4cfb033757b63ab0b128d01c58db76480956f9
Last commit: 7 days ago
Core tap JSON: 13 Jul 11:46 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit westmere
Clang: 14.0.3 build 1403
Git: 2.41.0 => /usr/local/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4.1-x86_64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: true
```

### Output of `brew doctor`

```shell
[No problems detected while running brew doctor.]
```

### Description of issue

The Homebrew version of Obsidian seems to be stuck at 1.3.5, whereas they've shipped a handful of cool features in 1.3.7, the latest version quite some time ago already.

### Relevant casks

obsidian

**Comment by ZhongRuoyu** [comment, marked as answer]
According to https://github.com/obsidianmd/obsidian-releases/blob/c7de3dd4f00db54584643bbec18f75623c7e09b9/desktop-releases.json: 1.3.7 is a beta release, and the latest version is still 1.3.5 at the moment. Beta software is not accepted in [Homebrew/homebrew-cask](https://github.com/Homebrew/homebrew-cask). (However, it can be submitted to [Homebrew/homebrew-cask-versions](https://github.com/Homebrew/homebrew-cask-versions); see ["Beta, Unstable, Development, Nightly, or Legacy"](https://docs.brew.sh/Acceptable-Casks#beta-unstable-development-nightly-or-legacy).)

Should there be a new *stable* release of Obsidian, you may open a PR. See ["Submit a new version of an existing cask"](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#submit-a-new-version-of-an-existing-cask).

**Reply by malikalimoekhamedov** [reply]
Lovely. Thank you.
